pra_request_tracker/pra_request_tracker/static/css/frontend.css

```diff
@@ -0,0 +1,3 @@
+pre {
+    white-space: pre-wrap;
+}
```

Comment:
It's always a little dicey putting rules on native elements, since it's changing a native behavior. But this change is so minimal (and arguably should be the default anyway), I have no problem with it.

Reply:
Good to know for the future though! This was the only place in the site we were using `pre`, so I thought it would be okay